src/main/java/edu/harvard/h2ms/service/UserServiceImpl.java

```diff
@@ -37,4 +45,13 @@ public User addUser(User user) {
 	public void deleteUser(Long id) {
 		userRepository.delete(id);
 	}
+	
+
+	@Override
+	public void save(User user) {
+		user.setPassword(bCryptPasswordEncoder.encode(user.getPassword()));
+//		user.setRoles(new HashSet<>(roleRepository.findAll()));

```

Comment:
Should this be deleted?

Reply:
That is a stub for role, so I'm not sure this is considered a dead code.  May I keep it?